### PR TITLE
Adds integration test

### PR DESCRIPTION
**Pull request by dmccown**

### CODE REVIEW

**Comment by dmccown** [comment]
I'm going to adjust this PR to ensure that integration tests run as part of Travis. I'll also annotate my change removing Ruby 2.6 from Travis CI.

**Comment by dmccown** [comment]
I'm going to have to back out on running the tests in travis. I don't think we can use vagrant in a travis ci run. We can investigate further in another issue.

File: README.md [hunk 1]
```diff
@@ -63,9 +72,48 @@ TODO: Write getting started
...
+Runs Rubucop syntax checker against code in this repository.
+
+```
+$ rake rubucop
+```
 
 
+#### Lint
+
+Runs Rubucop and Syntax checks against code in this repository. This is the default rake task.
```
Comment (TrevorBramble) [inline README.md]: Rubocop, syntax, Rake

File: README.md [hunk 2]
```diff
@@ -63,9 +72,48 @@ TODO: Write getting started
...
+Runs the Ruby syntax checker against code in this repository.
+
+```
+$ rake syntax
+```
+
+
+#### Rubucop
+
+Runs Rubucop syntax checker against code in this repository.
```
Comment (TrevorBramble) [inline README.md]: Rubocop

File: README.md [hunk 3]
```diff
@@ -63,9 +72,48 @@ TODO: Write getting started
...
+
+```
+$ rake rubucop
```
Comment (TrevorBramble) [inline README.md]: rubocop

File: README.md (hunk 2 above)
Comment (Padgett) [inline README.md]: typo `Rubucop` here and line 84

File: README.md (hunk 3 above)
Comment (Padgett) [inline README.md]: Also here

File: README.md (hunk 1 above)
Comment (Padgett) [inline README.md]: `Rubucop`

File: test/integration/verify/controls/habitat_service.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+control 'habitat_service' do
+  describe habitat_service(origin: 'core', name: 'nginx') do
+    it                     { should exist }
+    its('version')         { should cmp >= '1.15.5' }
+    its('topology')        { should eq 'standalone' }
+    its('update_strategy') { should eq 'none' }
+    its('deps.size')       { should eq 10 }
+    its('pkg_id')          { should match %r(core\/nginx\/\d+.\d+\.\d+\/\d{14}$) }
+    its('release')         { should cmp >= '20180608050617' }
```
Comment (Padgett) [inline test/integration/verify/controls/habitat_service.rb]: We're not pinning the `nginx` package to this version/release so tests will start failing if there's an update to that package on the depot

**Pull request merged**